includes/Checker/Runtime_Environment_Setup.php

```diff
@@ -223,11 +229,10 @@ public function can_set_up() {
 	/**
 	 * Installs WordPress, while providing tweaks to allow for early execution of the install process.
 	 *
-	 * @since 1.3.0
```

Comment:
Could you please undo all the unrelated changes like this?

Reply:
okay. It's the work of my IDE 🤦

Reply:
Done.